injector/helper.py

```
@@ -0,0 +1,67 @@
+        path = name + '.py'
+        pathsToCheck.append(path)
+    elif isinstance(node, ast.ImportFrom):
+        module = node.module.replace('.','/')
+        name = node.names[0].name.replace('.','/')
+        pathsToCheck.append(f"{module}/{name}.py".format(module,name))
+        pathsToCheck.append(f"{module}.py".format(module))
+    
+    validPaths = []
+    for path in pathsToCheck:
+        try:
+            open(path)
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)